plugins/Dashboard/javascripts/widgetMenu.js

```diff
@@ -13,7 +13,10 @@ function widgetsHelper() {
  *
  * @return {object} object containing available widgets
  */
-widgetsHelper.getAvailableWidgets = function () {
+widgetsHelper.getAvailableWidgets = function (callback) {
+    if(!callback) { // @deprecated
+        return this.getAvailableWidgetsSync();
```

Comment:
I reckon we don't have to be backwards compatible here, it's not really a public API or so unless some of our code still uses it